redaxo/src/core/fragments/core/form/search.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @var rex_fragment $this
+ * @psalm-scope-this rex_fragment
+ */
+
+$id = $clear = $class = $placeholder = '';
```

Comment:
```suggestion
$id = '';
```

Da die drei Variablen definitiv anschließend neu gesetzt werden, brauchen sie hier nicht gesetzt werden.